src/app/core/CommandLine.cs

```
@@ -66,27 +65,33 @@ public static RootCommand BuildRootCommand()
             OptionResult formatRes = result.Children.FirstOrDefault(c => c.Symbol.Name == "log-format") as OptionResult;
 
+            bool runLoop = result.Children.FirstOrDefault(c => c.Symbol.Name == "run-loop") is OptionResult runLoopRes && runLoopRes.GetValueOrDefault<bool>();
+            bool random = result.Children.FirstOrDefault(c => c.Symbol.Name == "random") is OptionResult randomRes && randomRes.GetValueOrDefault<bool>();
+            bool prom = result.Children.FirstOrDefault(c => c.Symbol.Name == "prometheus") is OptionResult promRes && promRes.GetValueOrDefault<bool>();
+            bool verbose = result.Children.FirstOrDefault(c => c.Symbol.Name == "verbose") is OptionResult verboseRes && verboseRes.GetValueOrDefault<bool>();
+            bool xml = result.Children.FirstOrDefault(c => c.Symbol.Name == "xml-summary") is OptionResult xmlRes && xmlRes.GetValueOrDefault<bool>();
+
             List<string> servers = serverRes.GetValueOrDefault<List<string>>();
             List<string> files = serverRes.GetValueOrDefault<List<string>>();
-            bool runLoop = runLoopRes.GetValueOrDefault<bool>();
-            int duration = durationRes.GetValueOrDefault<int>();
-            bool random = randomRes.GetValueOrDefault<bool>();
-            bool prom = promRes.GetValueOrDefault<bool>();
-            bool verbose = verboseRes.GetValueOrDefault<bool>();
-            bool xml = xmlRes != null && xmlRes.GetValueOrDefault<bool>();
-            LogFormat logFormat = formatRes.GetValueOrDefault<LogFormat>();
 
-            string errors = string.Empty;
+            int duration = 0;
+            LogFormat logFormat = LogFormat.Tsv;
+
+            try
+            {
+                duration = durationRes.GetValueOrDefault<int>();
+                logFormat = formatRes.GetValueOrDefault<LogFormat>();
+            }
+            catch
+            {
+            }
```

Is this intended to swallow the errors or should there be something in the catch?  If it is intended to catch and ignore, I would suggest a comment that explains that the intent is to ignore.

---

good catch - added comment on why we ignore